### ISSUE TITLE
dev-qt/qtpositioning == dev-qt/qtlocation ?

### ISSUE THREAD

**Issue by and3k** [shiznix/unity-gentoo#119]
Hi!

Is it possible that dev-qt/qtlocation should be renamed to the existing package of dev-qt/qtpositioning? I’m getting a lot of file collisions trying to install dev-qt/qtpositioning

Thanks,
Bela

Details:

```
 * Messages for package dev-qt/qtpositioning-5.4.2:

 * This package will overwrite one or more files that may belong to other
 * packages (see list below). You can use a command such as `portageq
 * owners / <filename>` to identify the installed package that owns a
 * file. If portageq reports that only one package owns a file then do
 * NOT file a bug report. A bug report is only useful if it identifies at
 * least two or more packages that are known to install the same file(s).
 * If a collision occurs and you can not explain where the file came from
 * then you should simply ignore the collision since there is not enough
 * information to determine if a real problem exists. Please do NOT file
 * a bug report at http://bugs.gentoo.org unless you report exactly which
 * two packages install the same file(s). See
 * http://wiki.gentoo.org/wiki/Knowledge_Base:Blockers for tips on how to
 * solve the problem. And once again, please do NOT file a bug report
 * unless you have completely understood the above message.
 * 
 * Detected file collision(s):
 * 
 *      /usr/lib64/libQt5Positioning.so.5.4.2
 *      /usr/lib64/libQt5Positioning.prl
 *      /usr/lib64/pkgconfig/Qt5Positioning.pc
 *      /usr/lib64/cmake/Qt5Positioning/Qt5Positioning_QGeoPositionInfoSourceFactoryPoll.cmake
 *      /usr/lib64/cmake/Qt5Positioning/Qt5PositioningConfigVersion.cmake
 *      /usr/lib64/cmake/Qt5Positioning/Qt5PositioningConfig.cmake
 *      /usr/lib64/qt5/plugins/position/libqtposition_positionpoll.so
 *      /usr/lib64/qt5/mkspecs/modules/qt_lib_positioning.pri
 *      /usr/lib64/qt5/mkspecs/modules/qt_lib_positioning_private.pri
 *      /usr/include/qt5/QtPositioning/QtPositioningDepends
 *      /usr/include/qt5/QtPositioning/QtPositioning
 *      /usr/include/qt5/QtPositioning/qtpositioningversion.h
 *      /usr/include/qt5/QtPositioning/QtPositioningVersion
 *      /usr/include/qt5/QtPositioning/qpositioningglobal.h
 *      /usr/include/qt5/QtPositioning/QNmeaPositionInfoSource
 *      /usr/include/qt5/QtPositioning/qnmeapositioninfosource.h
 *      /usr/include/qt5/QtPositioning/QGeoShape
 *      /usr/include/qt5/QtPositioning/qgeoshape.h
 *      /usr/include/qt5/QtPositioning/QGeoSatelliteInfoSource
 *      /usr/include/qt5/QtPositioning/qgeosatelliteinfosource.h
 *      /usr/include/qt5/QtPositioning/QGeoSatelliteInfo
 *      /usr/include/qt5/QtPositioning/qgeosatelliteinfo.h
 *      /usr/include/qt5/QtPositioning/QGeoRectangle
 *      /usr/include/qt5/QtPositioning/qgeorectangle.h
 *      /usr/include/qt5/QtPositioning/QGeoPositionInfoSourceFactory
 *      /usr/include/qt5/QtPositioning/qgeopositioninfosourcefactory.h
 *      /usr/include/qt5/QtPositioning/QGeoPositionInfoSource
 *      /usr/include/qt5/QtPositioning/qgeopositioninfosource.h
 *      /usr/include/qt5/QtPositioning/QGeoPositionInfo
 *      /usr/include/qt5/QtPositioning/qgeopositioninfo.h
 *      /usr/include/qt5/QtPositioning/QGeoLocation
 *      /usr/include/qt5/QtPositioning/qgeolocation.h
 *      /usr/include/qt5/QtPositioning/QGeoCoordinate
 *      /usr/include/qt5/QtPositioning/qgeocoordinate.h
 *      /usr/include/qt5/QtPositioning/qgeocircle.h
 *      /usr/include/qt5/QtPositioning/QGeoCircle
 *      /usr/include/qt5/QtPositioning/qgeoareamonitorsource.h
 *      /usr/include/qt5/QtPositioning/QGeoAreaMonitorSource
 *      /usr/include/qt5/QtPositioning/qgeoareamonitorinfo.h
 *      /usr/include/qt5/QtPositioning/QGeoAreaMonitorInfo
 *      /usr/include/qt5/QtPositioning/qgeoaddress.h
 *      /usr/include/qt5/QtPositioning/QGeoAddress
 *      /usr/include/qt5/QtPositioning/5.4.2/QtPositioning/private/qnmeapositioninfosource_p.h
 *      /usr/include/qt5/QtPositioning/5.4.2/QtPositioning/private/qlocationutils_p.h
 *      /usr/include/qt5/QtPositioning/5.4.2/QtPositioning/private/qgeoshape_p.h
 *      /usr/include/qt5/QtPositioning/5.4.2/QtPositioning/private/qgeorectangle_p.h
 *      /usr/include/qt5/QtPositioning/5.4.2/QtPositioning/private/qgeoprojection_p.h
 *      /usr/include/qt5/QtPositioning/5.4.2/QtPositioning/private/qgeopositioninfosource_p.h
 *      /usr/include/qt5/QtPositioning/5.4.2/QtPositioning/private/qgeolocation_p.h
 *      /usr/include/qt5/QtPositioning/5.4.2/QtPositioning/private/qgeocoordinate_p.h
 *      /usr/include/qt5/QtPositioning/5.4.2/QtPositioning/private/qgeocircle_p.h
 *      /usr/include/qt5/QtPositioning/5.4.2/QtPositioning/private/qgeoaddress_p.h
 *      /usr/include/qt5/QtPositioning/5.4.2/QtPositioning/private/qdoublevector3d_p.h
 *      /usr/include/qt5/QtPositioning/5.4.2/QtPositioning/private/qdoublevector2d_p.h
 *      /usr/include/qt5/QtPositioning/5.4.2/QtPositioning/private/qdeclarativegeolocation_p.h
 *      /usr/include/qt5/QtPositioning/5.4.2/QtPositioning/private/qdeclarativegeoaddress_p.h
 *      /usr/lib64/libQt5Positioning.so.5.4
 *      /usr/lib64/libQt5Positioning.so.5
 *      /usr/lib64/libQt5Positioning.so
 * 
 * Searching all installed packages for file collisions...
 * 
 * Press Ctrl-C to Stop
 * 
 * dev-qt/qtlocation-5.4.2:5::unity-gentoo
 *      /usr/include/qt5/QtPositioning/QGeoSatelliteInfoSource
 *      /usr/include/qt5/QtPositioning/QGeoShape
 *      /usr/include/qt5/QtPositioning/QNmeaPositionInfoSource
 *      /usr/include/qt5/QtPositioning/QtPositioning
 *      /usr/include/qt5/QtPositioning/QtPositioningDepends
 *      /usr/include/qt5/QtPositioning/QtPositioningVersion
 *      /usr/include/qt5/QtPositioning/qgeosatelliteinfosource.h
 *      /usr/include/qt5/QtPositioning/qgeoshape.h
 *      /usr/include/qt5/QtPositioning/qnmeapositioninfosource.h
 *      /usr/include/qt5/QtPositioning/qpositioningglobal.h
 *      /usr/include/qt5/QtPositioning/qtpositioningversion.h
 *      /usr/lib64/cmake/Qt5Positioning/Qt5PositioningConfig.cmake
 *      /usr/lib64/cmake/Qt5Positioning/Qt5PositioningConfigVersion.cmake
 *      /usr/lib64/cmake/Qt5Positioning/Qt5Positioning_QGeoPositionInfoSourceFactoryPoll.cmake
 *      /usr/lib64/libQt5Positioning.prl
 *      /usr/lib64/libQt5Positioning.so.5.4.2
 *      /usr/lib64/pkgconfig/Qt5Positioning.pc
 *      /usr/lib64/qt5/mkspecs/modules/qt_lib_positioning.pri
 *      /usr/lib64/qt5/mkspecs/modules/qt_lib_positioning_private.pri
 *      /usr/lib64/qt5/plugins/position/libqtposition_positionpoll.so
 * 
 * Package 'dev-qt/qtpositioning-5.4.2' NOT merged due to file
 * collisions. If necessary, refer to your elog messages for the whole
 * content of the above message.
```

**Comment by shiznix** [comment]
Fixed in commit 1103639cdc4365d830d8427323be18bc5cf238d8

Thanks for reporting :)

**Comment by and3k** [comment]
Thanks!
